provisioning/roles/nginx/templates/drupal7-site.j2

```diff
@@ -0,0 +1,61 @@
+{% extends "default-site.j2" %}
+
+{% block index %}
+    index index.php;
+{% endblock %}
+
+{% block extra %}
+    {{ super() }}
+
+    # Seems that the "index" block is not injected from the parent template, add it again here
+    index index.php;
```

Comment:
Did you correctly defined the `index` parameter in your `parameters,yml` AFAIL, the current system works.

Seeing how you are defining the the index block above, I think you are just misusing it. Maybe it needs more documentatrion.